src/__tests__/ProductView/QueryInterface.js

```diff
@@ -56,6 +56,21 @@ test('Handling save query', () => {
 });
 
```

Comment:
I think there is no other solution than figuring out each execution branch.
I did not find any coveralls option displaying it.